mobile/src/features/Legal/common/loadTextAsset.ts

```diff
@@ -0,0 +1,19 @@
+import {Asset} from 'expo-asset'
+
+export const loadTextAsset = async (assetModule: any): Promise<string> => {
+  try {
+    const asset = Asset.fromModule(assetModule)
+    await asset.downloadAsync()
+
+    if (asset.localUri) {
+      const response = await fetch(asset.localUri)
+      const text = await response.text()
+      return text
+    }
+
+    throw new Error('Failed to load asset')
+  } catch (error) {
+    console.error('Error loading text asset:', error)
```

Comment:
this will send to sentry if crash is enabled
```suggestion
    logger.error(error, {origin: 'loadTextAsset', error})
```